docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
+The `lint` command ensures that your definition file structure is valid (according to the OpenAPI Specification and your own custom rules, if any) and contains no errors. It also executes preprocessors and rules.
 
-:::success Tip
-To learn more about preprocessors and rules, refer to the [custom rules](../resources/custom-rules.md) page.
+:::warning Important
+Unlike the `bundle` command, `lint` doesn't execute decorators. To learn more about preprocessors and rules, see the [custom rules](../resources/custom-rules.md) page.
 :::
 
-## Usage
+## Options
 
-```bash
-openapi lint
-openapi lint <entrypoints>...
-openapi lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
-openapi lint [--generate-ignore-file] [--help]
-openapi lint --version
+Always the over-achiever, `lint` has many options which you can execute alone, or combine with others in a single command:
+
+```bash Standalone
+redocly lint
+redocly lint <entrypoints>...
+redocly lint --config=<path>
+redocly lint --extends=<value>
+redocly lint --format=<value>
+redocly lint --generate-ignore-file
+redocly lint --help
+redocly lint --max-problems=<n>
+redocly lint --skip-preprocessor=<preprocessor>
+redocly lint --skip-rule=<rule>,<rule>
+redocly lint --version
 ```
 
-## Options
+```bash Combined
+redocly lint [--max-problems=<n>] [--config=<path>] [--format=<value>]
+```
 
-Option                   | Type      | Required    | Default     | Description
--------------------------|:---------:|:------------:|:-----------:|------------
-`entrypoints`            | `array`   | no           | -           | Array of API definition filenames that need to be linted. See [the Entrypoints section](#entrypoints) for more options
-`--config`               | `string`  | no           | -           | Specify path to the [configuration file](#custom-configuration-file)
-`--extends`              | `array`   | no           | -           | [Extend a specific configuration](#extend-configuration) (defaults or config file settings)
-`--format`               | `string`  | no           | `codeframe` | Format for the output.<br />**Possible values:** `codeframe`, `stylish`, `json`, `checkstyle`
-`--generate-ignore-file` | `boolean` | no           | -           | [Generate ignore file](#generate-ignore-file)
-`--help`                 | `boolean` | no           | -           | Show help
-`--max-problems`         | `number`  | no           | 100         | Truncate output to display the specified [maximum number of problems](#max-problems)
-`--skip-preprocessor`    | `array`   | no           | -           | Ignore certain preprocessors. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--skip-rule`            | `array`   | no           | -           | Ignore certain rules. See the [Skip preprocessor or rule section](#skip-preprocessor-or-rule) below
-`--version`              | `boolean` | no           | -           | Show version number
+| Option | Type | Description |
```

Comment:
This matched the actual tool's help before. Now it doesn't.  This is also outdated in some places. I would recommend parity between the command's help and this content. A ticket can be opened to adjust the built in help, but I believe it should be in the standard declarative style all CLI tools I use adhere to.

Reply:
OK, I'll stick with the original table columns.